### PR TITLE
Arroyc/quickstart to Azure/azure-quickstart-templates/master

### PR DESCRIPTION
**Pull request by arroyc**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@arroyc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (arroyc). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: active-directory-new-domain/azuredeploy.json [hunk 1]
```diff
@@ -159,24 +167,20 @@
...
-    "adConfigurationFunction": "CreateADPDC.ps1\\CreateADPDC",
-    "apiVersion": "2015-06-15"
+    "adConfigurationFunction": "CreateADPDC.ps1\\CreateADPDC"
   },
   "resources": [
     {
       "type": "Microsoft.Storage/storageAccounts",
       "name": "[variables('storageAccountName')]",
-      "apiVersion": "[variables('apiVersion')]",
+      "apiVersion": "2015-06-15",
```
Comment (philon-msft) [inline active-directory-new-domain/azuredeploy.json]: Update Microsoft.Storage apiVersion to "2016-01-01" and replace its properties with:
      "sku": {
        "name": "Standard_LRS"
      },
      "kind": "Storage",

**Comment by gatneil** [comment]
Sorry for the delay; is this still something we want to merge in? If so, first step is to fix the merge conflicts

**Comment by philon-msft** [comment]
No, I already merged these changes in a separate PR.

**Pull request closed without merging**